helm/hpcc/values.schema.json

```diff
@@ -606,9 +606,30 @@
       	    "containers": {
               "type": "array",
               "description": "List of containers",
-              "items": { "type": "string" }
+              "items": {
+                "description": "storage container/file share (one item for each device)",
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "description": "Name of storage container/file share",
+                    "type" : "string"
+                  },
+                  "account" : {
+                    "description" : "Acccount Name (optional - may be provided in parent object instead)",
```

Comment:
trivial: I think I would word as "Account Name (optional - will default to account value in parent object if specified)"